views/index.html

```diff
@@ -32,6 +32,135 @@ <h1 class="name">Hector Torres</h1>
+                </section>
+            </article>
+        </section>
+        <section class="about intro">
+            <article class="about-intro">
+                <h1 class="name">About</h1>
+                <p class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                    incididunt ut
+                    labore
+                    et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                    aliquip
+                    ea
+                    eiusmod</p>
+                <button class="btn" type="button"><span>Get mt CV<img class="link-img" src="../images/ic_download.svg"
+                            alt="project link"></span></button>
+            </article>
+            <article class="about-cards">
+                <div class="about-card1 about-1">
+                    <h2 class="about-header">Languages</h2>
+                    <ul class="list">
+                        <li class="item"><span class="list-text">JavaScript (ES5/ES6)</span></li>
+                        <li class="item"><span class="list-text">Ruby</span></li>
+                        <li class="item"><span class="list-text">HTML</span></li>
+                        <li class="item"><span class="list-text">CSS</span></li>
+                    </ul>
+                    <img class="about-img1" src="../images/num.png" alt="numer sing">
+                </div>
+                <div class="about-card2 about-2">
+                    <h2 class="about-header">Frameworks</h2>
+                    <ul class="list">
+                        <li class="item"><span class="list-text">Bootstrap</span></li>
+                        <li class="item"><span class="list-text">Ruby on rails</span></li>
+                        <li class="item"><span class="list-text">RSpec</span></li>
+                        <li class="item"><span class="list-text">Capibara</span></li>
+                        <li class="item"><span class="list-text">Selenium</span></li>
+                    </ul>
+                    <img class="about-img2" src="../images/Group_18.png" alt="vector figure">
+                </div>
+                <div class="about-card about-3">
+                    <h2 class="about-header">Skills</h2>
+                    <ul class="list">
+                        <li class="item"><span class="list-text">Codekit</span></li>
+                        <li class="item"><span class="list-text">Codepen</span></li>
+                        <li class="item"><span class="list-text">GitHub</span></li>
+                        <li class="item"><span class="list-text">GitLAb</span></li>
+                        <li class="item"><span class="list-text">Terminal</span></li>
+                    </ul>
+                    <img class="about-img3" src="../images/ic_skills.svg" alt="ic skill">
+                </div>
+            </article>
+        </section>
```

Comment:
- [ ] Please ensure that you use the `display: flex;` property to position items within this page. This so as to improve on the page's responsiveness.